astilectron.go

```diff
@@ -182,6 +185,48 @@ func (a *Astilectron) provision() error {
 	return a.provisioner.Provision(ctx, a.options.AppName, runtime.GOOS, runtime.GOARCH, *a.paths)
 }
 
+// function to create TCP/Unix socket connection
+func (a *Astilectron) establishConnection() (err error) {
```

Comment:
Could you rename this function to `listen` instead ?

Reply:
name was modified as per suggestion